dist/uPlot.esm.d.ts

```diff
@@ -276,6 +276,8 @@ declare namespace uPlot {
 		max?: number,
 	}
 
+	export type SpanGap = [number, number]
```

Comment:
let's make this `export type DataGap = [number, number];`

Reply:
Done.